docs/components/badge.md

```diff
@@ -77,32 +80,40 @@ Using the `custom` appearance in conjunction with `color` and `background-color`
 | `--KBadgeDangerBorder`      |                               |
 | `--KBadgeDangerBackground`  |                               |
 
-An example of making the badges more like pills
+An example of making the badges more rad
 
 > Note: We are scoping the overrides to a wrapper in this example
 
 <template>
   <div class="KBadge-wrapper">
-    <KBadge appearance="danger">HIGH</KBadge>
+    <KBadge appearance="danger">DANGER - RADIOACTIVE MATERIAL</KBadge>
   </div>
 </template>
 
 ```vue
 <template>
   <div class="KBadge-wrapper">
-    <KBadge appearance="danger">HIGH</KBadge>
+    <KBadge appearance="danger">DANGER</KBadge>
   </div>
 </template>
 
 <style>
 .KBadge-wrapper {
-  --KBadgeBorderRadius: 50px;
+  --KBadgeBorderRadius: 3px;
```

Comment:
This coloring makes my eyes bleed a little bit - maybe maybe a color combo a little less painful on the eyes?

Reply:
![image](https://user-images.githubusercontent.com/67973710/107409355-13a61b80-6ada-11eb-842b-1701bbf64716.png)

Reply:
BRUTALISM

Reply:
Made it more corporate in https://github.com/Kong/kongponents/pull/288/commits/e60c754e076146508d4c910a8cd775f7b8841dd9